src/TraceEvent/BPerf/BPerfEventSource.cs

```diff
@@ -521,7 +542,7 @@ private unsafe int ProcessBTLInner(int eof)
                 fixed (byte* uncompressedBufferPtr = &this.uncompressedBuffer[0])
```

Comment:
With this change, do you still need `uncompressedBufferPtr` and `compressedBufferPtr`?  It looks like you pass the objects into the delegate and then let the delegate pin if it needs to.

Reply:
removed compressedBufferPtr.   but uncompressedBufferPtr  still needed because in next five lines use it for DeserializeEventRecord